src/alert_methods/SCP/alert.orig

```diff
@@ -0,0 +1,90 @@
+#!/bin/sh
+# Copyright (C) 2016-2018 Greenbone Networks GmbH
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+
+# Escalator method script: SCP.
+
+USERNAME=$1
+HOST=$2
+DEST=$3
+KNOWN_HOSTS=$4
+PRIVATE_KEY_FILE=$5
+PASSWORD_FILE=$6
+REPORT_FILE=$7
+<<<<<<< Updated upstream
```

Comment:
Looks like this entire file was accidentally added.

Reply:
Yes, I've removed it in c421becab2ad86d506365fe36f3d4fa1d363aa7c